drivers/eaton-pdu-marlin-mib.c

```diff
@@ -175,27 +175,28 @@ static info_lkp_t marlin_input_type_info[] = {
 	{ 0, NULL }
 };
 
+static info_lkp_t marlin_outlet_group_phase_info[] = {
+	{ 0, "unknown", NULL, NULL, NULL, NULL }, /* unknown     */
+	{ 1, "1", NULL, NULL, NULL, NULL },       /* singlePhase */
+	{ 2, "1-N", NULL, NULL, NULL, NULL },     /* phase1toN   */
+	{ 3, "2-N", NULL, NULL, NULL, NULL },     /* phase2toN   */
+	{ 4, "3-N", NULL, NULL, NULL, NULL },     /* phase3toN   */
+	{ 5, "1-2", NULL, NULL, NULL, NULL },     /* phase1to2   */
+	{ 6, "2-3", NULL, NULL, NULL, NULL },     /* phase2to3   */
+	{ 7, "3-1", NULL, NULL, NULL, NULL },     /* phase3to1   */
+	{ 0, NULL, NULL, NULL, NULL, NULL }
```

Comment:
Something in this table upset the DMF parser on Travis builds, will try to reproduce. Maybe some `is None` is missing in our Pythonware.

````
3523 I: Parsed /home/travis/build/42ity/nut/drivers/eaton-pdu-marlin-mib.c OK
3524 I: CALL Visitor():
3525 Traceback (most recent call last):
3526  File "/home/travis/build/42ity/nut/scripts/DMF/jsonify-mib.py", line 572, in <module>
3527    v.visit (node)
3528  File "/usr/local/lib/python2.7/dist-packages/pycparser/c_ast.py", line 120, in visit
3529    return visitor(node)
3530  File "/home/travis/build/42ity/nut/scripts/DMF/jsonify-mib.py", line 319, in visit_Decl
3531    self._visit_info_lkp_t (node)
3532  File "/home/travis/build/42ity/nut/scripts/DMF/jsonify-mib.py", line 205, in _visit_info_lkp_t
3533    debug("fun_l2s.name : %s" % (fun_l2s.name))
3534 AttributeError: 'Cast' object has no attribute 'name'
3535 ERROR: Could not parse '/home/travis/build/42ity/nut/drivers/eaton-pdu-marlin-mib.c' into 'eaton-pdu-marlin-mib.dmf'
3536       You can inspect a copy of the intermediate result in 'eaton-pdu-marlin-mib.json.tmp', 'eaton-pdu-marlin-mib.dmf.tmp' and 'eaton-pdu-marlin-mib_TEST.c'
3537 make[2]: *** [dmfsnmp/eaton-pdu-marlin-mib.dmf] Error 1
````

Reply:
strange, I did not had that one on my side